src/components/add_book.js

```diff
@@ -20,36 +27,50 @@ const AddBook = () => {
     if (title.trim().length === 0 || author.trim().length === 0) {
       return;
     }
-    dispatch(asyncAdd({
-      item_id: v4(), title, author, category: 'Not classified yet',
-    }));
+    dispatch(
+      asyncAdd({
+        item_id: v4(),
+        title,
+        author,
+        category: 'Not classified yet',
+      }),
+    );
     setFormState({ title: '', author: '' });
   };
   return (
-    <form
-      onSubmit={handleSubmit}
-      style={{
-        width: '50%', margin: '10px auto', display: 'flex', flexDirection: 'column',
-      }}
-    >
-      <input
-        type="text"
-        name="title"
-        placeholder="Book title"
-        value={title}
-        onChange={handleChange}
-      />
-      <input
-        type="text"
-        name="author"
-        placeholder="Book author"
-        value={author}
-        onChange={handleChange}
-      />
-      <button type="submit" className="input-submit">
-        Submit
-      </button>
-    </form>
+    <>
+      <h4
+        style={{
+          marginTop: '2%',
+          marginBottom: '2%',
+          marginLeft: '10%',
+        }}
+      >
+        ADD NEW BOOK
+      </h4>
+
+      <form onSubmit={handleSubmit} style={formStyle}>
+        <input
+          className="title-input"
+          type="text"
+          name="title"
+          placeholder="Book title"
+          value={title}
+          onChange={handleChange}
+        />
+        <input
+          className="author-input"
+          type="text"
+          name="author"
+          placeholder="Book author"
+          value={author}
+          onChange={handleChange}
+        />
+        <button type="submit" className="input-submit">
+          Submit
+        </button>
+      </form>
+    </>
```

Comment:
- [ ] The form is different from the original. It is ok if you add author input but please add category input also.
- [ ] Please align the form to the book list.
- [ ] Please change the update progress button style like in the original.

![image](https://user-images.githubusercontent.com/38635158/174399893-57548124-daec-4fd5-adc3-3951b3fdf433.png)
__
Original
![image](https://user-images.githubusercontent.com/38635158/174399951-7c63df7b-a7a0-43e5-957b-9825f3187c53.png)